contracts/BuyBackBurner.sol

```diff
@@ -26,6 +26,8 @@ contract BuyBackBurner {
     bytes32 public constant BUY_BACK_BURNER_PROXY = 0xc6d7bd4bd971fa336816fe30b665cc6caccce8b123cc8ea692d132f342c4fc19;
     // L1 OLAS Burner address
     address public constant OLAS_BURNER = 0x51eb65012ca5cEB07320c497F4151aC207FEa4E0;
+    // Max allowed price deviation for TWAP pool values (10%) in 1e18 format
```

Comment:
Max allowed deviation is moved here. Please carefully check the deviation. We suggest to keep no less than 10%.